Engine/ac/dynobj/cc_dynamicarray.h

```diff
@@ -93,7 +93,9 @@ namespace DynamicArrayHelpers
     // Create array of managed strings from strings that exists somewhere
     DynObjectRef CreateStringArray(const std::vector<const char*>);
     // Create array of managed strings from script strings
-    DynObjectRef CreateStringArray(std::vector<ScriptString::Buffer> &&);
+    DynObjectRef CreateStringArrayFromBuffers(std::vector<ScriptString::Buffer> &&);
```

Comment:
There's really no need to have different name, it was fine as a function overload.

Reply:
I had to remove the overload because the linter I use was not picking up the overload kept telling me this function was unused, I wasn't comfortable with it.